lms/static/sass/views/_teams.scss

```diff
@@ -346,6 +346,106 @@
     }
   }
 
+  .team-profile {
+    .discussion-module {
+      width: 60%;
+      @include float(left);
+    }
+    .team-detail {
+      width: 30%;
+      @include float(left);
+      @include margin($baseline, 0, $baseline, $baseline + $baseline/2);
+
+      div, h4 {
+        margin-bottom: ($baseline/4);
+      }
+      @extend %t-copy-sub1;
```

Comment:
If this belongs to team-detail it would be good to move up to the earlier list of properties. currently it is split by the div / h4 styling.